src/utils/rpc_errors.ts

```diff
@@ -57,6 +69,8 @@ function walkSubtype(errorObj, schema, result, typeName) {
         }
         return walkSubtype(error, schema, result, errorTypeName);
     } else {
+        // TODO: is this the right thing to do?
```

Comment:
unfortunately I don't know, as I have no understanding of what are possible responses from nearcore side. 

also I think we should completely remove everything related to error schema https://github.com/near/near-api-js/issues/331

Reply:
@bowenwang1996 I feel like you may know a bit about the nearcore errors. Would you mind giving us your two cents when you get a moment this week?

Reply:
an error only has `kind` if it is an action error but there are many other errors out there